app/src/main/java/edu/gatech/spacebarz/buzzshelter/model/Shelter.java

```diff
@@ -1,115 +1,116 @@
 package edu.gatech.spacebarz.buzzshelter.model;
 
-import android.location.Location;
-
-/**
- * Created by mayaholikatti on 2/19/18.
- */
-
 public class Shelter {
+    public Shelter() {};
```

Comment:
We should add the comment back here that this constructor is only for firebase

Reply:
Not sure what the comment was, I just moved my Shelter class from the CSV uploader over.

Reply:
Ok, I can add it back in

Reply:
Its the same comment that is in the UserInfo class